docs/ref/contrib/postgres/fields.txt

```diff
@@ -17,6 +17,27 @@ ArrayField
     <ArrayField.size>`. ``ArrayField`` can be nested to store multi-dimensional
     arrays.
 
+    .. warning::
+
+        When using a model's instance methods, it is not recommended to use
```

Comment:
The discussion of "model instance methods" seems a bit out of place as we are talking about field defaults which are specified on the model field, not on instance methods. Also, I think the Marc's suggestion about how to use non-empty defaults (with lambda's) should be included.